Firebase/InstanceID/FIRInstanceIDCheckinService.m

```diff
@@ -179,6 +188,8 @@ - (void)checkinWithExistingCheckin:(FIRInstanceIDCheckinPreferences *)existingCh
 
 - (void)stopFetching {
   [self.session invalidateAndCancel];
+  // The session cannot be reused after invalidation. Dispose it to prevent accident reusing.
+  self.session = nil;
```

Comment:
Is it a one time use thing then? Or should `session` be re-initialized at some point? Hoping @chliangGoogle has more context for where/how this is used.

Reply:
lol I had the same question, although this has been like this for a while, so I'm not sure why it suddenly crash updating to 5.17.0. Could it also relate to the fact that 5.17.0 has a weird version of IID and we haven't figured out the exact cause of it.

I think it might worth to wait until we figure out why developers still see old behavior when updating to 5.17.0/5.18.0 ( #2438).